example/server/api/index.js

```diff
@@ -1,32 +1,22 @@
 const Router = require('express').Router;
-const environmentID = 'uCDQzKWgejrutqSYYsKWen';
-const flagsmith = require('../../../');
-const NodeCache = require('node-cache');
+const { Flagsmith } = require('../../../sdk');
 
-flagsmith.init({
-    environmentID
-    // this is an example of a user-defined cache
-    /*cache: new NodeCache({
```

Comment:
@dabeeeenster as an FYI this removes the work in https://github.com/Flagsmith/flagsmith-nodejs-client/pull/16 that allows adding in a form of caching such as redis or node-cache.

I think that we should make sure this is in for V2 since it was requested

Reply:
Separated out to here https://github.com/Flagsmith/flagsmith-nodejs-client/issues/31